frontend/src/static/js/main.js

```diff
@@ -62,14 +58,11 @@ window.app = new window.Vue({
 
     deactivateTabs() {
       this.isTabAuthorship = false;
-      this.isTabIssues = false;
     },
 
     updateTabAuthorship(obj) {
       this.deactivateTabs();
-      this.tabRepo = obj.repo;
-      this.tabAuthor = obj.author;
-      this.tabAuthorName = obj.name;
+      this.tabAuthorship = { ...obj };
```

Comment:
if you want to support extensions, perhaps this should be given a generic name instead of `tabAuthorship`?

Reply:
oh, the idea is to have one object for each module, so instead of `tabAuthor`, `tabAuthorName` and `tabRepo` for the authorship tab, we have just one `tabAuthorship` object that is used to passed all the details into the authorship module.

Reply:
but in future, you're going to use that tab to display commits etc as well isn't it? so it wouldn't be right to call it `tabAuthorship` anymore.